tutorials/01_introductory/sample_plots.py

```diff
@@ -58,8 +59,8 @@
 You can add arbitrary paths in matplotlib using the
```

Comment:
nah, just leave it as is then and we can make it someone else's problem. I think the arbitrary is important.